### PR TITLE
add hijri-date-picker

### PR DESCRIPTION
**Pull request by abublihi**

### CODE REVIEW

File: README-ar.md [hunk 1]
```diff
@@ -37,7 +37,8 @@
 * [Hijri.js](https://github.com/xsoh/Hijri.js) -  أداة بسيطة للتقويم الهجري على الجافاسكربت.
 * [moment-hijri](https://github.com/xsoh/moment-hijri) -  إضافة لـmoment.js تقوم بتزويدك بالتفويم الهجري داخل مومينت.
 * [moment-jalaali](https://github.com/jalaali/moment-jalaali) - إضافة لـmoment.js تقوم بتزويدك بالتفويم الهجري الشمسي داخل مومينت.
-
+* [hijri-date-picker](https://github.com/abublihi/hijir-date-picker) - مكون بسيط لإختيار التاريخ في الويب،
+تم عمله بواسطة [رياكت](https://reactjs.org).
```
Comment (xsoh) [inline README-ar.md]: Please Add new line here

File: README.md [hunk 2]
```diff
@@ -35,6 +35,7 @@
 * [Hijri.js](https://github.com/xsoh/Hijri.js) -  Tool for the Islamic calender (Hijri) in Javascript.
 * [moment-hijri](https://github.com/xsoh/moment-hijri) -  A Hijri calendar (Based on Umm al-Qura calculations) plugin for moment.js
 * [moment-jalaali](https://github.com/jalaali/moment-jalaali) - A Jalaali (Jalali, Persian, Khorshidi, Shamsi) calendar system plugin for moment.js.
+* [hijri-date-picker](https://github.com/abublihi/hijir-date-picker) - A simple and reusble react component for hijir date pickre.
```
Comment (xsoh) [inline README.md]: This should be:
* [hijri-date-picker](https://github.com/abublihi/hijir-date-picker) - A simple and reusable Hijri date picker react component.

**Comment by abublihi** [comment]
تم، شكراً

**Comment by xsoh** [comment]
يعطيك العافية

**Pull request merged**